### ISSUE TITLE
macOS App.. please add version information to the info.plist

### ISSUE THREAD

**Issue by macmule** [unetbootin/unetbootin#316]
https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html#//apple_ref/doc/uid/20001431-111349

TIA!

**Comment by core-code** [comment]
yes! both CFBundleVersion and CFBundleShortVersionString are missing
